packages/react-core/src/components/Modal/ModalBox.tsx

```diff
@@ -36,10 +42,15 @@ export const ModalBox: React.FunctionComponent<ModalBoxProps> = ({
     className={css(
       styles.modalBox,
       className,
+      position === 'top' && styles.modifiers.alignTop,
       variant === 'large' && styles.modifiers.lg,
       variant === 'small' && styles.modifiers.sm,
       variant === 'medium' && styles.modifiers.md
     )}
+    style={{
+      ...(positionOffset && { '--pf-c-modal-box--m-align-top--spacer': positionOffset }),
```

Comment:
i noticed that when i pass in positionOffset, it isn't just moved down vertically, but the width also shrinks, is that supposed to happen?

Reply:
@mcoker ^

Reply:
@jschuler @kmcfaul does this have the latest version of core? That should have been fixed in https://github.com/patternfly/patternfly/pull/3481

Reply:
It did not, now it does :)